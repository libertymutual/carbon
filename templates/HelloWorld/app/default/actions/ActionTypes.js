// APP ACTIONS

// AUTH ACTIONS
export const LOGIN = 'login';
export const LOGOUT = 'logout';
export const AUTH_UPDATE_AUTH_URL = 'auth_update_auth_url'
export const AUTH_UPDATE_CLIENT_ID = 'auth_update_client_id'
export const AUTH_UPDATE_CLIENT_SECRET = 'auth_update_client_secret'
export const AUTH_UPDATE_CONFIG_NAME = 'auth_update_config_name'
export const AUTH_UPDATE_PASSWORD = 'auth_update_password'
export const AUTH_UPDATE_USERNAME = 'auth_update_username'
export const AUTH_UPDATE_VALIDATOR_ID = 'auth_update_validator_id'

// COUNTER ACTIONS
export const INCREMENT = 'increment';
export const DECREMENT = 'decrement';

// BIOMETRIC ACTIONS
export const UPDATE_USERNAME = 'update_username';
export const UPDATE_PASSWORD = 'update_password';
export const SAVE_CREDENTIALS = 'save_credentials';
export const DELETE_CREDENTIALS = 'delete_credentials';
