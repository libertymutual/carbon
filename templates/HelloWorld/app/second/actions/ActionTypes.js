// APP ACTIONS

// HELLO WORLD ACTIONS
export const HELLO = 'hello';
export const GOODBYE = 'goodbye';

// COUNTER ACTIONS
export const INCREMENT = 'increment';
export const DECREMENT = 'decrement';
