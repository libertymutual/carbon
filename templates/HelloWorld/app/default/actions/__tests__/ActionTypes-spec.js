import * as ActionTypes from '../ActionTypes';

describe('ActionTypes tests', () => {

  test('LOGIN action type returns login', () => {
    expect(ActionTypes.LOGIN).toMatchSnapshot();
  });

  test('LOGOUT action type returns logout', () => {
    expect(ActionTypes.LOGOUT).toMatchSnapshot();
  });

  test('AUTH_UPDATE_AUTH_URL action type returns auth_update_auth_url', () => {
    expect(ActionTypes.AUTH_UPDATE_AUTH_URL).toMatchSnapshot();
  });

  test('AUTH_UPDATE_CLIENT_ID action type returns auth_update_client_id', () => {
    expect(ActionTypes.AUTH_UPDATE_CLIENT_ID).toMatchSnapshot();
  });

  test('AUTH_UPDATE_CLIENT_SECRET action type returns auth_update_client_secret', () => {
    expect(ActionTypes.AUTH_UPDATE_CLIENT_SECRET).toMatchSnapshot();
  });

  test('AUTH_UPDATE_CONFIG_NAME action type returns auth_update_config_name', () => {
    expect(ActionTypes.AUTH_UPDATE_CONFIG_NAME).toMatchSnapshot();
  });

  test('AUTH_UPDATE_PASSWORD action type returns auth_update_password', () => {
    expect(ActionTypes.AUTH_UPDATE_PASSWORD).toMatchSnapshot();
  });

  test('AUTH_UPDATE_USERNAME action type returns auth_update_username', () => {
    expect(ActionTypes.AUTH_UPDATE_USERNAME).toMatchSnapshot();
  });

  test('AUTH_UPDATE_VALIDATOR_ID action type returns auth_update_validator_id', () => {
    expect(ActionTypes.AUTH_UPDATE_VALIDATOR_ID).toMatchSnapshot();
  });

  test('INCREMENT action type returns increment', () => {
    expect(ActionTypes.INCREMENT).toMatchSnapshot();
  });

  test('DECREMENT action type returns decrement', () => {
    expect(ActionTypes.DECREMENT).toMatchSnapshot();
  });

  test('UPDATE_USERNAME action type returns update_username', () => {
    expect(ActionTypes.UPDATE_USERNAME).toMatchSnapshot();
  });

  test('UPDATE_PASSWORD action type returns update_password', () => {
    expect(ActionTypes.UPDATE_PASSWORD).toMatchSnapshot();
  });

  test('SAVE_CREDENTIALS action type returns save_credentials', () => {
    expect(ActionTypes.SAVE_CREDENTIALS).toMatchSnapshot();
  });

  test('DELETE_CREDENTIALS action type returns delete_credentials', () => {
    expect(ActionTypes.DELETE_CREDENTIALS).toMatchSnapshot();
  });
});
