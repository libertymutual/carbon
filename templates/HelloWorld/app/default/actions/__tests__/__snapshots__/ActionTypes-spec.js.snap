// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionTypes tests AUTH_UPDATE_AUTH_URL action type returns auth_update_auth_url 1`] = `"auth_update_auth_url"`;

exports[`ActionTypes tests AUTH_UPDATE_CLIENT_ID action type returns auth_update_client_id 1`] = `"auth_update_client_id"`;

exports[`ActionTypes tests AUTH_UPDATE_CLIENT_SECRET action type returns auth_update_client_secret 1`] = `"auth_update_client_secret"`;

exports[`ActionTypes tests AUTH_UPDATE_CONFIG_NAME action type returns auth_update_config_name 1`] = `"auth_update_config_name"`;

exports[`ActionTypes tests AUTH_UPDATE_PASSWORD action type returns auth_update_password 1`] = `"auth_update_password"`;

exports[`ActionTypes tests AUTH_UPDATE_USERNAME action type returns auth_update_username 1`] = `"auth_update_username"`;

exports[`ActionTypes tests AUTH_UPDATE_VALIDATOR_ID action type returns auth_update_validator_id 1`] = `"auth_update_validator_id"`;

exports[`ActionTypes tests DECREMENT action type returns decrement 1`] = `"decrement"`;

exports[`ActionTypes tests DELETE_CREDENTIALS action type returns delete_credentials 1`] = `"delete_credentials"`;

exports[`ActionTypes tests INCREMENT action type returns increment 1`] = `"increment"`;

exports[`ActionTypes tests LOGIN action type returns login 1`] = `"login"`;

exports[`ActionTypes tests LOGOUT action type returns logout 1`] = `"logout"`;

exports[`ActionTypes tests SAVE_CREDENTIALS action type returns save_credentials 1`] = `"save_credentials"`;

exports[`ActionTypes tests UPDATE_PASSWORD action type returns update_password 1`] = `"update_password"`;

exports[`ActionTypes tests UPDATE_USERNAME action type returns update_username 1`] = `"update_username"`;
